save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/FossGraphView.kt

```diff
@@ -147,6 +164,20 @@ val fossGraph: FC<FossGraphViewProps> = FC { props ->
                         }
                     }
                 }
+                column("delete", "") { cellProps ->
+                    Fragment.create {
+                        td {
+                            div {
+                                className = ClassName("d-flex justify-content-end")
+                                buttonBuilder(faTrashAlt, style = "") {
+                                    setDeleteProject(value = cellProps.row.original)
+                                    setIsDeleteProject(true)
```

Comment:
Why do you need `isDeleteProject?` It seems that making `deleteProject` nullable gives you everything you need

Reply:
And still it seems that we have `vulnerability` and `project` so:
1.  either is not null -> delete it
2. none is not null -> do not show modal
3. both is not null -> throw IllegaStateException

Reply:
Deleted it